mono/mini/mini-amd64.c

```diff
@@ -1166,6 +1166,12 @@ mono_arch_get_native_call_context_ret (CallContext *ccontext, gpointer frame, Mo
 	MonoEECallbacks *interp_cb = mini_get_interp_callbacks ();
 	CallInfo *cinfo;
 
+	if (0) {
+done:
```

Comment:
This avoids goto around initialization -- which is illegal in C++ -- and is alternate maybe alien solution. It is often the smaller diff. 
Eventually we might have "early return" instead but not  for a while.